Books/client/src/features/misc/routes/Landing.tsx

```diff
@@ -0,0 +1,80 @@
+import digitalMinimalism from '../../../assets/images/digital_minimalism.webp';
+import readingNook from '../../../assets/images/readingNook.svg';
+import './Landing.scss';
+
+export const Landing = () => {
+  return (
+    <>
+      <nav className="flex justify-end">
+        <button className="secondary-btn">Sign In</button>
+        <button className="primary-btn">Sign Up</button>
+      </nav>
+      <section className="flex justify-between section-container">
+        <div className="hero__text">
+          <h1 className="h1 heading">Find your next best book and read it all day.</h1>
+          <p className="paragraph">
+            At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium
+            voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint
+            occaecati cupiditate non provident
+          </p>
```

Comment:
That paragraph should been different color?

Reply:
Just fixed this to match the Figma Design, is it okay for you?